src/vote_program.rs

```diff
@@ -10,7 +10,7 @@ use std::collections::VecDeque;
 use transaction::Transaction;
 
 // Upper limit on the size of the Vote State
-pub const MAX_STATE_SIZE: usize = 1024;
+pub const MAX_STATE_SIZE: usize = 512;
```

Comment:
A couple words in the comment justifying 512 or 1k for a Vote state would help.   I see `MAX_VOTE_HISTORY` below, does that have anything to do with MAX_STATE_SIZE:?

Reply:
Why "nearest power of two ..."?   Just curious why it can't be exactly `MAX_VOTE_HISTORY * sizeof(Vote) + sizeof(Pubkey)`?